libkbfs/folder_branch_ops.go

```diff
@@ -2267,7 +2255,13 @@ func (fbo *folderBranchOps) createEntryLocked(
 	// Passthrough journal writes temporarily.
 	if excl == WithExcl {
 		if jServer, err := GetJournalServer(fbo.config); err == nil {
-			wasEnabled, err := jServer.Disable(ctx, fbo.id())
```

Comment:
I looked over this PR, and the added complexity to support a relatively uncommon case makes me sad. What do you think of having `Disable` return a specific error when the journal is non-empty, checking for it here, and doing a `Wait` and retrying `Disable` until that error passes?

Since block archiving and deletion is the only thing that happens, it seems like this will work before timing out enough to not be a problem. What do you think?